### DISCUSSION TITLE
I can't create my account - Your uploaded proof has already been used previously. (?)

### DISCUSSION THREAD

**Discussion by Unaid8** [community/community#84076] [GitHub Education]
### Hi everyone,

When I uploaded my student ID, I got hit with this message: "Your uploaded proof has already been used previously.” But I've never done this before, and this account is the one and only I've got on GitHub. What should I do?
<img width="1288" alt="Captura de pantalla 2024-01-03 a las 23 02 00" src="https://github.com/community/community/assets/134620800/049b7770-55da-4f75-af68-3d47c880d9a6">

**Comment by codesmith-emmy** [comment]
Please visit [GitHub Support](https://support.github.com/contact) and open a ticket. The issue would be looked into and resolved immediately. 🙂

**Reply by samus-aran** [reply]
Hi @codesmith-emmy 

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.

In this particular situation it appears it can be resolved without the need for a support team member.

**Reply by codesmith-emmy** [reply]
Okay. Thank you for the clarification.

**Comment by samus-aran** [comment]
Hi @Unaid8 👋 

It appears that you had consecutively submitted applications previously. I would suggest you take another photo of your ID so that it is different from the previous submission. 

Did you use the live camera option?
